src/Renderer/HtmlRenderer.php

```diff
@@ -491,16 +493,33 @@ public function createServerInformationLink(ServerRequestInterface $request): st
     }
 
     /**
-     * Returns the name of the throwable instance.
+     * Returns string representation of the throwable name.
      *
-     * @return string The name of the throwable instance.
+     * @param Throwable $throwable The throwable.
+     *
+     * @return string The throwable name.
      */
     public function getThrowableName(Throwable $throwable): string
     {
         $name = $throwable::class;
 
         if ($throwable instanceof FriendlyExceptionInterface) {
             $name = $throwable->getName() . ' (' . $name . ')';
+        } else {
+            // Check if the exception class has FriendlyException attribute
+            try {
+                $reflectionClass = new ReflectionClass($throwable);
+                if (class_exists('Yiisoft\FriendlyException\Attribute\FriendlyException')) {
```

Comment:
Use full class name notation without quotes
```suggestion
                if (class_exists(\Yiisoft\FriendlyException\Attribute\FriendlyException::class)) {
```